packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts

```diff
@@ -101,7 +102,7 @@ const login = (account: string) => (
 ) => {
   const useGSN = getState().env['gsn']?.enabled;
   const provider = getWeb3Provider(windowRef);
-  const networkId = windowRef.web3.currentProvider.networkVersion;
+  const networkId = windowRef.web3?.currentProvider?.networkVersion || getState().env['networkId'];
```

Comment:
I recommend getting the network ID from `eth_chainId` (https://eips.ethereum.org/EIPS/eip-695).  This is the standard (for almost a year) way of getting the chain ID from the Ethereum provider.